### PR TITLE
Issue #9: Add a helper for ddev to run task easily

### PR DESCRIPTION
**Pull request by justafish**
…includes

## Testing Instructions
- [x] Run `composer create-project drupal/recommended-project drainpipe-test`
- [x] Install [ddev](https://ddev.readthedocs.io/en/stable/)
- [x] In drainpipe-test run `ddev config`
- [x] In drainpipe-test run `ddev start`
- [x] Inside the web container, check out this repo and branch
- [x] Add this to the repositories key in composer.json of drainpipe-test:
```
        {
            "type": "path",
            "version": "dev-main",
            "url": "/path/to/drainpipe",
            "options": {
                "symlink": false
            }
        },
```
- [x] Run `ddev composer require lullabot/drainpipe:dev-main`
- [x] Run `ddev task --list`

### CODE REVIEW

**Comment by rabbitlair** [comment]
When running `ddev task --list` command, I get the following error:

```
$ ddev task --list
/mnt/ddev_config/commands/web/task: line 7: ./vendor/bin/task: cannot execute binary file: Exec format error
Failed to run task --list: exit status 126
```

**Review by rabbitlair — APPROVED** [review]
Peer reviewed with @justafish, it's working fine.

File: scaffold/ddev/task-command.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (deviantintegral) [inline scaffold/ddev/task-command.sh:1]: TIL'ed, this is an awesome ddev feature! 👏

File: src/TaskfileInstallerPlugin.php [hunk 2]
```diff
@@ -142,4 +144,18 @@ private function installGitignore(): void
             }
         }
     }
+
+    /**
+     *
+     */
+    private function installDdevCommand(): void
+    {
+        if (file_exists('./.ddev/config.yaml')) {
```
Comment (deviantintegral) [inline src/TaskfileInstallerPlugin.php:153]: I wonder if we should document this somewhere. I imagine an existing project pulling in drainpipe first, then later pulling in ddev, and not knowing that running `composer update lullabot/drainpipe` would be a benefit. Perhaps in the readme for now?

**Pull request merged**